pizzapi/menu.py

```diff
@@ -71,30 +71,30 @@ def display(self):
         def print_category(category, depth=1):
             indent = "  " * (depth + 1)
             if len(category.products) + len(category.subcategories) > 0:
-                print indent + category.name
+                print(indent + category.name)
                 for subcategory in category.subcategories:
                     print_category(subcategory, depth + 1)
                 for product in category.products:
-                    print indent + "  [%s]" % product.code, product.name
-        print "************ Coupon Menu ************"
+                    print(indent + "  [%s]" % product.code, product.name)
+        print("************ Coupon Menu ************")
         print_category(self.root_categories['Coupons'])
-        print "\n************ Preconfigured Menu ************"
+        print("\n************ Preconfigured Menu ************")
         print_category(self.root_categories['PreconfiguredProducts'])
-        print "\n************ Regular Menu ************"
+        print("\n************ Regular Menu ************")
         print_category(self.root_categories['Food'])
 
     # TODO: Find more pythonic way to format the menu
     # TODO: Format the menu after the variants have been filtered
     # TODO: Return the search results and print in different method
     # TODO: Import fuzzy search module or allow lists as search conditions
     def search(self, **conditions):
-        max_len = lambda x: 2 + max(len(v[x]) for v in self.variants.values())
-        for v in self.variants.itervalues():
+        max_len = lambda x: 2 + max(len(v[x]) for v in list(self.variants.values()))
+        for v in self.variants.values():
             v['Toppings'] = dict(x.split('=', 1) for x in v['Tags']['DefaultToppings'].split(',') if x)
-            if all(y in e.get(x, '') for x, y in conditions.iteritems()):
-                print v['Code'],
-                print v['Name'],
-                print '$' + v['Price'],
-                print v['SizeCode'],
-                print v['ProductCode'],
-                print v['Toppings']
+            if all(y in v.get(x, '') for x, y in conditions.items()):
+                print(v['Code'], end=' ')
+                print(v['Name'], end=' ')
+                print('$' + v['Price'], end=' ')
+                print(v['SizeCode'], end=' ')
+                print(v['ProductCode'], end=' ')
```

Comment:
I like your idea of supporting Python 3, but we shouldn't remove support for Python 2.

Please set the first line of the files that use print to:
```python
from __future__ import print_function
```

Reply:
ah good call.  ill get that updated